bittorrent/player.go

```diff
@@ -536,6 +536,22 @@ func (btp *BTPlayer) chooseFile() (int, error) {
 			choices = append(choices, candidate)
 		}
 
+		if btp.episode > 0 {
+			foundMatches := int64(0)
+			lastMatched := int(-1)
```

Comment:
You don't need to cast a value, and `int64` just for a counter? This should work just fine:
```
var lastMatched int
var foundMatches int
```